src/components/PageSettingsModal.jsx

```diff
@@ -44,29 +47,13 @@ export default class PageSettingsModal extends Component {
       } = this.props;
 
       // get settings data from backend
-      const settingsResp = await axios.get(`${process.env.REACT_APP_BACKEND_URL}/sites/${siteName}/settings`, {
-        withCredentials: true,
-      });
+      const settingsResp = await axios.get(`${process.env.REACT_APP_BACKEND_URL}/sites/${siteName}/settings`);
```

Comment:
We could consider refactoring out the url as a constant, since it's used in quite a few places here

Reply:
Resolved with [0282467]([url](https://github.com/isomerpages/isomercms-frontend/pull/168/commits/0282467d67cefa8204d76287bf819d0a0aefbe26))